test/vaadin-dialog_draggable-resizable-test.html

```diff
@@ -552,6 +553,11 @@
           draggable.opened = true;
         });
 
+        (!MOBILE ? it : it.skip)('should prevent default of the touchstart when dragged on desktop', () => {
```

Comment:
nit: Prefer having all tests run regardless of the device used for testing where feasible. Example: [here](https://github.com/vaadin/vaadin-grid/blob/9dec44691fd54aedb8fc75aa01e7f5ae7dd8a263/test/drag-and-drop.html#L372-L382) the `_ios` property is manually set to a specific value at the beginning of the tests. Not a blocker.

Reply:
I wasn't sure if I should introduce extra property just for testing. 
Added now and updated